test/regression/cases/ddot_logs/datadog-agent/datadog.yaml

```diff
@@ -11,6 +11,9 @@ logs_config:
   logs_no_ssl: true
   force_use_http: true
 
-dd_url: http://127.0.0.1:9092
-telemetry.enabled: true
-telemetry.checks: '*'
+dd_url: http://127.0.0.1:9091
+process_config:
+  process_dd_url: http://localhost:9093
```

Comment:
Fyi, we used to have a bug where `localhost` didn't resolve in target containers. This may need to be `127.0.0.1` to work; it's possible that's no longer an issue, which would be interesting to know.

Reply:
According to logs, it seems to work fine: https://app.datadoghq.com/logs?query=job_id%3A0e94ca24-dad5-4f20-8d5f-2f5355288322%20container_name%3Atarget%20experiment%3Addot_logs%209093&agg_m=count&agg_m_source=base&agg_t=count&event=AwAAAZZKrKHEaQq8cwAAABhBWlpLcktPT0FBQ25mdzBqcl9rbUpnREsAAAAkMDE5NjRhYjEtYjg1My00ZTlhLTg1NTEtYjg5ZDA4Zjk1YWQwAA0XjQ&fromUser=true&index=single-machine-performance%2Csingle-machine-performance-low-prio%2Csingle-machine-performance-target-logs&messageDisplay=inline&refresh_mode=paused&storage=hot&stream_sort=desc&viz=stream&from_ts=1745009304978&to_ts=1745010495540&live=false